+neurostim/stimulus.m

```diff
@@ -258,11 +260,11 @@ function addDiodeFlasher(s,varargin)
                 for sp=1:size(specs,1)
                     s.(specs{sp,2}) = specs{sp,3};
                 end
+                localizeParms(s,true);  % Update those loc parameters that change within a trial

```

Comment:
This was missing for updates to localized RSVP parms

Reply:
I am still not completely on top of how the localizeParms solution works, but is this potentially a costly step to do each new rsvp stimulus? Params that are in the rsvp are known to change in trial from the start, so can they be set as such with your `changesInTrial()` approach?

Reply:
Most stimuli will not have localized parms and for them this adds no processing time. 

For stimuli that have loc parms (or that use some of the stimulus base class loc parms, like I did), the code warns the user who can then add those parameters (as instructed at run time) to the changesInTrial set. 

Automating this is possible, but given the rarity not a priority.

Reply:
But wouldn't it still make sense to include `changesInTrial()` in `addRSVP()` in this case? The purpose of the RSVP is to change params mid-trial, so it seems odd to me to require the user to manually add it to their script. the loc properties are only used in plugins where there were likely already performance issues.

https://github.com/klabhub/neurostim/blob/bd792eaa8c483990a03cdaf0a872e344394f9720/%2Bneurostim/stimulus.m#L177

Reply:
Most RSVPs will not use loc variables (because very few variables are localized). If an RSVP does use a loc, the user will be warned and the user can get better performance by adding those to changesInTrial, but even if the user ignores this, no harm (Except a slight drop in performance) is done. 

Automating this is possible but not trivial, by which I mean that I am not going to do this now.